### DISCUSSION TITLE
Fullscreen Pictures in iOS App?

### DISCUSSION THREAD

**Discussion by extrafein** [photoview/photoview#1005] [Q&A and Troubleshooting]
First, I love PhotoView 🥇 

### Question
Is there a way to show the pictures in the iOS App in fullscreen mode?
I'll like the option to see ExifData, but there are use cases I only want to see the picture in all it's greatness.

Thx in advance for help.

**Comment by kkovaletp** [comment]
Thanks for your question!
I thought that the project for the iOS app also has a Discussions section and tried to move this discussion there, but found that it is not possible)
So, let's keep it here) Maybe someone uses the iOS app and can answer your question)
